src/AddTodoForm.jsx

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable react/prop-types */
+export default function AddTodoForm(props) {
+  const handleAddTodo = (event) => {
+    event.preventDefault();
+    var todoTitle = event.target.elements.title.value;
```

Comment:
for a variable that doesn't change, we prefer to use a constant here with the `const` keyword.